site/cypress/e2e/Cypress-Gradeable/rainbow_grading.spec.js

```diff
@@ -52,6 +198,27 @@ skipOn(Cypress.env('run_area') === 'CI', () => {
         });
     });
 });
+const checkCheckbox = (testId) => {
+    cy.get(testId).as('checkbox');
+    cy.get('@checkbox').check();
+    cy.get('@checkbox').should('be.checked');
+    cy.get('@checkbox').uncheck();
+    cy.get('@checkbox').should('not.be.checked');
+    cy.get('@checkbox').check();
+    cy.get('@checkbox').should('be.checked');
+};
+const checkTextbox = (testId, expectedInitial, input) => {
+    cy.get(testId).as('textbox');
+    cy.get('@textbox').should('have.value', expectedInitial);
+    cy.get('@textbox').type('{selectAll}{backspace}');
```

Comment:
Is this different than 'cy.get('@textbox').clear()' ?

Reply:
Changed in [c2a784f](https://github.com/Submitty/Submitty/pull/10726/commits/c2a784fa1c3b52180dad04ced8851c71474a2f81)